.github/workflows/start-self-hosted-runner/action.yml

```diff
@@ -0,0 +1,32 @@
+name: Start self hosted EC2 runner
+
+inputs:
+    aws-region:
```

Comment:
why take these as arguments, instead of taking them from the environment in this script?

Reply:
You can't directly use secrets in called actions